modules/gitops-release-notes-1-5-0.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-5-0_{context}"]
+= Release notes for {gitops-title} 1.5.0
+
+[role="_abstract"]
+{gitops-title} 1.5.0 is now available on {product-title} 4.8 and above.
+
+[id="new-features-1-5-0_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* This enhancement upgrades Argo CD to version *2.3.3*. link:https://issues.redhat.com/browse/GITOPS-1708[GITOPS-1708]
+
+* This enhancement upgrades Dex to version *2.30.3*. link:https://issues.redhat.com/browse/GITOPS-1850[GITOPS-1850]
+
+* This enhancement upgrades Helm to version *3.8.0*. link:https://issues.redhat.com/browse/GITOPS-1709[GITOPS-1709]
+
+* This enhancement upgrades Kustomize to version *4.4.1*. link:https://issues.redhat.com/browse/GITOPS-1710[GITOPS-1710]
+
+* This enhancement upgrades Application Set to version *0.4.1*.
+
+* With this update, a new channel that is named *latest* has been added. This channel provides the latest release of the {gitops-title}. For GitOps v1.5.0, the Operator is pushed to *gitops-1.5*, *latest* channel, and the existing *stable* channel. From GitOps v1.6, all of the latest releases will be pushed only to the *latest* channel and not the *stable* channel. link:https://issues.redhat.com/browse/GITOPS-1791[GITOPS-1791]
+
+* With this update, the new CSV adds the `olm.skipRange: '>=1.0.0 <1.5.0'` annotation. As a result, all of the previous release versions are skipped. The Operator upgrades to v1.5.0 directly. link:https://issues.redhat.com/browse/GITOPS-1787[GITOPS-1787]
+
+* With this update, the Operator updates the Red Hat Single Sign-On (RH-SSO) to version v7.5.1, including the following enhancements:
+
+** You can log in to Argo CD using the {product-title} credentials including the `kube:admin` credential.
+** The RH-SSO supports and configures Argo CD instances for Role-based Access Control (RBAC) using {product-title} groups.
+** The RH-SSO honors the `HTTP_Proxy` environment variables. You can use the RH-SSO as an SSO for Argo CD running behind a proxy.
++
+link:https://issues.redhat.com/browse/GITOPS-1330[GITOPS-1330]
+
+* With this update, a new `.host` URL field is added to the `.status` field of the Argo CD operand. When a route or ingress is enabled with the priority given to route, the new URL field displays the route. If no URL is provided from the route or ingress, the `.host` field is not displayed.
++
+When the route or ingress is configured, but the corresponding controller is not set up properly and is not in the `Ready` state or does not propagate its URL, the value of the `.status.host` field in the operand indicates as `Pending` instead of displaying the URL. This affects the overall status of the operand by making it `Pending` instead of `Available`. link:https://issues.redhat.com/browse/GITOPS-654[GITOPS-654]
```

Comment:
```suggestion
When the route or ingress is configured, but the corresponding controller is not set up properly and is not in the `Ready` state or does not propagate its URL, the value of the `.status.host` field in the operand is indicated as `Pending` instead of displaying the URL. This affects the overall status of the operand by making it `Pending` instead of `Available`. link:https://issues.redhat.com/browse/GITOPS-654[GITOPS-654]
```